draft-ietf-quic-http.md

```diff
@@ -1908,13 +1920,13 @@ A client can discard responses that it cannot process.
 
 ### CONNECT Issues
 
-The CONNECT method can be used to create disproportionate load on a proxy,
-since stream creation is relatively inexpensive when compared to the creation
-and maintenance of a TCP connection.  A proxy might also maintain some resources
-for a TCP connection beyond the closing of the stream that carries the CONNECT
+The CONNECT method can be used to create disproportionate load on a proxy, since
+stream creation is relatively inexpensive when compared to the creation and
+maintenance of a TCP connection.  A proxy might also maintain some resources for
+a TCP connection beyond the closing of the stream that carries the CONNECT
 request, since the outgoing TCP connection remains in the TIME_WAIT state.
-Therefore, a proxy cannot rely on QUIC stream limits alone to control the
-resources consumed by CONNECT requests.
+Therefore, a proxy might delay increasing the QUIC stream limits to account for
+the resources consumed by CONNECT requests.
```

Comment:
The original text existed to note that accounting for streams as requests is insufficient in the case that a CONNECT request is used.  Delay does no good.  What we might instead say is that any proxy that supports CONNECT might choose to limit the number of concurrent CONNECT streams it accepts (and account for the higher cost of those streams when considering overall resource allocation).

Reply:
It looks like the original text did two things: note that CONNECT streams are more expensive than request streams (as you note here), but also that resources used for CONNECT streams might linger even after the HTTP stream is closed (the point I remarked upon where the properties of HTTP/3 and HTTP/2 diverge qualitatively).

Reply:
Hm, that's a different reading that I'd initially taken.  I'll try to rework the text a bit more.